src/main/scala/ai/privado/dataflow/Dataflow.scala

```diff
@@ -96,7 +105,11 @@ class Dataflow(cpg: Cpg, statsRecorder: StatsRecorder) {
               _.tag.nameExact(Constants.catLevelOne).valueExact(CatLevelOne.SOURCES.name),
               _.tag.nameExact(InternalTag.OBJECT_OF_SENSITIVE_CLASS_BY_MEMBER_NAME.toString)
             )
-          sinks.reachableByFlows(firstLevelSources).l
+
+          sinks
+            .reachableByFlows(firstLevelSources)
+            .l
+            .flatMap(addOriginalSourceToDataflowPath)
```

Comment:
I have a suggestion, to add the original source to a path in the `dataflowCache.finalDataflow`. Because, in the current approach we are generating original source for all, but we limit this to dataflow which will be final.

Reply:
Would It make sense to get this while exporting? 🤔